soroban-env-host/src/test/stellar_asset_contract.rs

```diff
@@ -322,6 +323,30 @@ impl StellarAssetContractTest {
     }
 }
 
+fn check_event_name(test: &StellarAssetContractTest, e: HostEvent, contract: &Address, name: &str) {
+    assert!(e.event.type_ == ContractEventType::Contract && e.event.contract_id.is_some());
+
+    let address = ScAddress::Contract(e.event.contract_id.clone().unwrap());
+    let id = test.host.add_host_object(address).unwrap();
+    assert!(
+        test.host
+            .obj_cmp(contract.as_object().to_val(), id.to_val())
+            .unwrap()
+            == 0
+    );
+
+    match e.event.body {
```

Comment:
nit: this could be simplified to just `let xdr::ContractEventBody::V0(contract_event_v0) = e.event.body;`

Reply:
Fixed